src/main/resources/swagger.api/examples/ResourceIdsResult.sample

```diff
@@ -0,0 +1,14 @@
+{
+  "ids": [
+    {
+      "id": "e7e7f4a4-ff8e-48f4-ae8e-25770b7ab4b9"
+    },
+    {
+      "id": "e60063f6-db50-4a2a-93f6-6b27ab0420f2"
+    },
+    {
+      "id": "5fab345a-30a4-4aee-8667-1e33d4627ce9"
+    }
+  ],
+  "totalRecords": 3
+}
```

Comment:
Would an array of String work? The extra "id" seems a bit redundant, or maybe you just want to be flexible?

Reply:
The same contract is used for request, so I've decided to keep it for UI to make it easier to integrate: 

 `.../record-bulk/ids?limit=2147483647&query=(keyword all "book") sortby title"`